neurips2022/track2/submission/policy.py

```diff
@@ -6,7 +6,10 @@
 import pickle
 import torch
 from torchvision import transforms
-from model_IL import MainNet
+try:
+    from submission.model_IL import MainNet
+except:
+    from model_IL import MainNet
```

Comment:
Why do we have this import issue? 

If possible, consider adding the appropriate parent folder to the search path to resolve the imports more elegantly.